src/App.js

```diff
@@ -9,12 +12,12 @@ import Footer from './components/Footer';
 class App extends Component {
     render() {
         return (
-            <React.Fragment>
```

Comment:
why react fragment is replaced with a pane?

Reply:
I tried to do a blur effect upon navbar hover. So I tried to use all components inside our main page, contain them inside a CSS container and experiment with whether it could be done or not.